### ISSUE TITLE
Error "Cannot add duplicate edge" building a router db

### ISSUE THREAD

**Issue by alecava58** [itinero/routing#219]
Hi all,

building a routerdb of Italy I get a "Cannot add duplicate edge" exception.
I had an old PBF file but I tried also with the lastest.

I'm using IDP built with the latest alpha of Itinero.

is there something that I can check or try?

![image](https://user-images.githubusercontent.com/1788993/44775698-8d4e2700-ab76-11e8-93d4-e792679a55e2.png)

**Comment by xivk** [comment]
Testing this now...

**Comment by xivk** [comment]
What profiles? car, bicycle, pedestrian?

**Comment by alecava58** [comment]
Car

alessandro

Il dom 2 set 2018, 16:14 Ben Abelshausen <notifications@github.com> ha
scritto:

**Comment by xivk** [comment]
Did you use the build from the FTP or build yourself? I just tested with the latest build of IDP, after updating Itinero to latest, built locally.

I ran this:

`dotnet run -c release --read-pbf europe-latest.osm.pbf --pr --create-routerdb vehicle=car -
-write-routerdb europe.routerdb`

I test this with `italy`, `belgium`, `germany`, `europe` and `north-america` and they all work. There is something different between what you are doing and what I'm testing.
